server/src/main/java/com/google/musicanalysis/site/YoutubeServlet.java

```diff
@@ -40,14 +40,78 @@ protected void doGet(HttpServletRequest req, HttpServletResponse res)
 
         var httpClient = HttpClient.newHttpClient();
         var youtubeReq = HttpRequest.newBuilder(youtubeUri)
-                .header("Authorization", "Bearer " + accessToken.toString())
+                .header("Authorization", "Bearer " + accessToken)
                 .header("Accept", "application/json")
                 .GET()
                 .build();
 
         // get response to Youtube API 
         var youtubeRes = httpClient.sendAsync(youtubeReq, BodyHandlers.ofString()).join();
```

Comment:
nit: don't use `var` for variables where the type is not obvious

Reply:
oops haha, will fix